web/modules/weather_data/src/Service/AlertTrait.php

```diff
@@ -227,6 +227,13 @@ public function getAlerts($grid, $point, $self = false, $now = false)
 
         $alerts = AlertUtility::sort($alerts);
 
+        // The API's alert ids are just too long for use
+        // in the DOM. Let's loop through and simply
+        // assign an index value
+        for ($i = 0; $i < count($alerts); $i++) {
+            $alerts[$i]->alertId = $i + 1;
+        }
+
```

Comment:
Agreed, those IDs are bonkers, and this is probably smarter than using the Twig index property. 😅 

We'll revisit this in #862 as well. One quirk of using indices is that the links in the table can be different colors: visited anchors will be black, unvisited ones will be blue. (cc: @igorkorenfeld on whether that's the desired behavior). But the unintended effect is that a completely different alert will have the exact same link in the future, so it can be misleading to users about what they have or haven't seen.

Reply:
That is a really good point. Maybe we can get a smaller id by doing something like a hash of the timestamp, the alert type, and the first applicable forecast/county zone code